Specimen/Specimen-BoneMarrowAspiration-Example.json

```diff
@@ -0,0 +1,44 @@
+{
+  "resourceType": "Specimen",
+  "id": "Specimen-BoneMarrowAspiration-Example",
+  "identifier": [
+    {
+      "system": "https://fhir.add.nhs.uk/Id/specimenId",
+      "value": "RR812434"
+    }
+  ],
+  "accessionIdentifier": {
+    "system": "https://www.leedsth.nhs.uk/path",
+    "value": "RR8123745"
+  },
+  "status": "available",
+  "type": {
+    "coding": [
+      {
+        "system": "http://snomed.info/sct",
+        "code": "396997002",
+        "display": "Specimen from bone marrow obtained by aspiration"
+      }
+    ]
+  },
+  "subject": {
+    "reference": "Patient/Patient-PatrickSammy-Example"
+  },
+  "receivedTime": "2023-08-13T09:00:00Z",
+  "request": [
+    {
+      "reference": "ServiceRequest-NonWGSTestOrderForm-HaemOncology-Example"
+    }
+  ],
+  "collection": {
+    "extension": [
+      {
+        "url": "http://hl7.org/fhir/StructureDefinition/specimen-specialHandling",
```

Comment:
These should be observations referencing the sample (or a note on the sample if you want to keep this unstructured). The sample itself is not a high risk of infection.

Reply:
Comment 1 & Comment 2: Further clarifications required before merging.